invenio_vocabularies/contrib/affiliations/schema.py

```diff
@@ -36,7 +37,20 @@ class AffiliationSchema(BaseVocabularySchema, ModePIDFieldVocabularyMixin):
             )
         )
     )
-    name = SanitizedUnicode(required=True)
+    name = SanitizedUnicode(
+        required=True, validate=validate.Length(min=1, error=_("Name cannot be blank."))
+    )
+    country = SanitizedUnicode()
+    country_name = SanitizedUnicode()
+    location_name = SanitizedUnicode()
```

Comment:
(nit) Question: Why are using country, country_name, location_name? isn't it more intuitive to use country_code, country, location?

Reply:
For legacy compatibility with funders https://github.com/inveniosoftware/invenio-vocabularies/blob/b0255d543037248ce096b66a8ffbd678339bb4ac/invenio_vocabularies/contrib/funders/schema.py#L46. I agree country_code makes more sense, but not sure it's worth the effort to rename in funders.